### PR TITLE
Add latency metrics for the Event publishers

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Add latency metrics for the Event publishers to the messaging server.

### CODE REVIEW

**Review by berkanercan — APPROVED** [review]
Looking good

File: components/event-publisher-proxy/pkg/handler/http/handler_test.go [hunk 1]
```diff
@@ -346,6 +371,8 @@ func TestHandlerTimeout(t *testing.T) {
 	_ = resp.Body.Close()
 	if http.StatusInternalServerError != resp.StatusCode {
 		t.Fatalf("Test failed, want status code:%d but got:%d", http.StatusInternalServerError, resp.StatusCode)
+	} else {
```
Comment (sayanh) [inline components/event-publisher-proxy/pkg/handler/http/handler_test.go:374]: (nit): else is redundant

**Review by sayanh — APPROVED** [review]

File: components/event-publisher-proxy/pkg/handler/http/handler_test.go (hunk 1 above)
Comment (marcobebway) [reply components/event-publisher-proxy/pkg/handler/http/handler_test.go]: Removed.

**Comment by marcobebway** [comment]
Image updates should be done in this [PR](https://github.com/kyma-project/kyma/pull/10975).

**Pull request merged**